### ISSUE TITLE
RouteDescriberTrait is now internal, which triggers a deprecation

### ISSUE THREAD

**Issue by gnutix** [nelmio/NelmioApiDocBundle#1673]
Since I've migrated to Symfony 5.1, I have the following deprecation on every page : 

```
The "Nelmio\ApiDocBundle\RouteDescriber\RouteDescriberTrait" trait is considered internal. It may change without further notice. You should not use it from "App\UI\Rest\Nelmio\CommandQueryDescriber".
```

As RouteDescriberTrait (just as ModelRegistryAwareTrait) are listed in examples to create your own RouteDescriberInterface implementation, I don't see why one would be set as internal while the other does not (and found no explanation on this repo on why the change happened in the first place).

How should I deal with this ?

**Comment by GuilhemN** [comment]
Hi!

This trait is internal since its creation, I guess symfony 5.1 now takes it into account.

> As RouteDescriberTrait (just as ModelRegistryAwareTrait) are listed in examples to create your own RouteDescriberInterface implementation

What examples are you talking about? Are you referring to the route describers bundled in this repository?

> How should I deal with this ?

I think we can remove the `@internal` tag, this is only here because no one expressed the need until now, but I see no issue about making this trait public.
Would you like to submit a PR?

**Comment by gnutix** [comment]
I could. On which branch should I do the change?

**Comment by GuilhemN** [comment]
Please target 3.x :) I'll merge it in master then/

**Comment by gnutix** [comment]
I know it is trivial, but I don't seem to find the right moment to do it. If someone wants to pick it, they can.

**Comment by GuilhemN** [comment]
Fixed in #1718, thank you!
